radiation/python/radlw/lwrad_gt4py.py

```diff
@@ -0,0 +1,1800 @@
+import gt4py
+import os
+import sys
+import time
+import numpy as np
+import xarray as xr
+import gt4py.gtscript as gtscript
+
+sys.path.insert(0, "..")
+from radlw_param import a0, a1, a2, nbands, nrates, ipat, delwave
+from stencils_gt4py import *
+from util import (
+    view_gt4py_storage,
+    compare_data,
+    create_storage_from_array,
+    create_storage_zeros,
+    create_storage_ones,
+    loadlookupdata,
+)
+from config import *
+
+sys.path.append(SERIALBOX_DIR + "/python")
+import serialbox as ser
+
+ddir = "../../fortran/data/LW"
+serializer = ser.Serializer(ser.OpenModeKind.Read, ddir, "Generator_rank0")
+savepoints = serializer.savepoint_list()
+
+ddir2 = "../../fortran/radlw/dump"
+serializer2 = ser.Serializer(ser.OpenModeKind.Read, ddir2, "Serialized_rank0")
```

Comment:
is only rank 0 tested? this is probably sufficient, though with all the if conditions sometimes testing a higher volume of data can help catch issues. Have you tried the other ranks? We can leave this code as-is for now, but am curious.

Reply:
Yes only rank 0. I can try with other ranks, good point about the if statements